TypeScript/Grid/Editing.md

```diff
@@ -768,7 +768,7 @@ After the template elements are converted to JS controls using actionComplete ev
 
 Users can start editing by clicking a cell and typing data into it. Edited cell will be marked while navigating to next cell or any other row, so that you know which fields or cells has been edited. Set [`editMode`](https://help.syncfusion.com/api/js/ejgrid#members:editsettings-editmode "editMode") as `batch` to enable batch editing.
 
-N> Refer the KB [link](http://www.syncfusion.com/kb/3016/how-to-suppress-grid-confirmation-messages# "link") for "How to suppress grid confirmation messages" in batch mode.
+N> Refer the KB [link](https://support.syncfusion.com/kb/article/3065/how-to-suppress-grid-confirmation-messages?isInternalRefresh=False "link") for "How to suppress grid confirmation messages" in batch mode.
```

Comment:
Check this link
<img width="685" alt="image" src="https://github.com/syncfusion-content/typescript-docs/assets/95272306/12ebab53-6552-43b9-9d95-67875c147e41">

Reply:
Resolved